en/micro-integrator/mkdocs.yml

```diff
@@ -427,6 +427,7 @@ nav:
       - 'Best Practices' : 'references/best-Practices.md'
       - 'Troubleshooting JMS' : 'references/troubleshooting-jms.md'
       - 'Error Codes' : 'references/error_handling.md'
+      - 'WSO2 Micro Integrator vs WSO2 ESB' : 'references/comparisong-mi7-ei6xx.md'
```

Comment:
This need to be changed as WSO2 Micro Integrator vs WSO2 EI 6.x family ESB Profile